kpi/views/browser_kpis.view.lkml

```diff
@@ -15,6 +15,30 @@ view: browser_kpis {
     hidden: yes
   }
 
+  dimension: active_hours_sum {
```

Comment:
I'm hiding these because I want the KPI view to be focused solely on the user count. if people want to dig into these dimensions they can use the unified metrics explore.